index.xml

```diff
@@ -577,6 +594,7 @@ The signature header and signature would be generated as:
     <figure>
      <artwork>
 Signature: keyId="rsa-key-1",algorithm="rsa-sha256",
+date=1402170695, expires=1402170699,
```

Comment:
I first proposed an internal standard based on RFC3339 but a couple of security guys suggested to use timestamps to avoid parsing issues. RFC3339 and ISO should theoretically manage timezone offset too.